website/blog/2022/03/28/release-apache-apisix-2.13.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Apache APISIX 2.13.0 Is Released, Bringing LTS and New Features"
+authors:
+  - name: "Zexuan Luo"
+    title: "Author"
+    url: "https://github.com/spacewander"
+    image_url: "https://avatars.githubusercontent.com/u/4161644?v=4"
+  - name: "Yilin Zeng"
+    title: "Technical Writer"
+    url: "https://github.com/yzeng25"
+    image_url: "https://github.com/yzeng25.png"
+keywords: 
+- Apache APISIX
+- 2.13.0
+- Release Notes
+- API Gateway
+- LTS version
+description: Today, the Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
+tags: [Release]
+---
+
+> Today, the Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```

Comment:
```suggestion
> Apache APISIX community is bringing a new LTS release, 2.13.0, which not only provides more stable performance, but also supports more observability and service discovery plugins, and a better multilingual development system.
```

Reply:
Apache APISIX community has rolled out a new LTS version, 2.13.0. This version not only delivers more stable performance, but also supports more observability and service discovery plugins, as well as a more robust multilingual development system.